onboarding/sprint-rotation.md

```diff
@@ -5,49 +5,39 @@ description: What is the sprint rotation and how does it work?
-- Pending time, the engineer can get their dev environment set up with any additional systems.
-- Pending time, the engineer can browse the team's documentation and current sprint projects.
-
-_Tech lead/PM responsibilities:_
-- Meet with the new engineer to discuss the focus for the upcoming sprint and help them get acquainted or onboarded with any team processes or systems.
-- Add the engineer to all team meetings _just for that sprint_. This includes standups, knowledge shares, retros, and the sprint planning for the assigned sprint (which will likely occur the week before).
-- Make sure the backlog has a healthy number of `good-first-issue` tickets that can be picked up during the sprint.
-- Let the team know that the sprint rotation will be happening so they are prepared to pair on tasks and feel comfortable explicitly scheduling that if necessary.
-- Invite the new engineer to necessary slack user groups and/or google groups for the team.
-
-#### During the sprint
-- Engineer will pair on critical sprint tasks with members of the team.
-- Engineer will work on non-blocking small tasks (i.e. those [tagged with](https://artsyproduct.atlassian.net/issues/?jql=labels%20%3D%20good-first-issue) `good-first-issue`), pairing with their mentor or members of the team if necessary.
-
+Once an engineer has gone through general onboarding and done some basic set-up, they'll spend the following few
+sprints rotating through a variety of engineering teams.
+
+### Why rotate?
+
+We want engineers to be familiar with the full breadth of products, systems, and people at Artsy, but at a
+comfortable pace. Rather than immediately joining a team for the long-term and focusing on that team's priorities,
+rotations allow engineers to tour the teams, get to know other engineers, and touch a variety of layers and
+systems. These rotations may influence the choice of team an engineer joins, or may just provide valuable context
+to their future work.
+
+### How do sprint rotations work?
+
+Managers will arrange for new engineers to join a sequence of 3 or more teams for 1 sprint each. Emphasis should be
+on encountering a variety of teams and technology rather than having impact in a familiar space. Following these
+rotations, the engineer will join a team on a more stable basis.
+
+#### Managers' responsibilities
+
+- Taking into account other rotating engineers and individual teams' schedules, managers add each rotation to the
+  [engineering onboarding calendar](https://calendar.google.com/calendar/b/1?cid=YXJ0c3ltYWlsLmNvbV9jc2RocTFnNHNyMmhhY2Nvc3RmMTA1bTEwc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)🔒.
+- For each rotation, managers should nominate a pairing partner from the team. This other engineer can help the new
+  engineer get set up, include them in their own sprint work, and be a primary point of contact for context about
+  tickets and projects.
+- Collaborate with each team's technical lead to ensure a healthy backlog of
```

Comment:
I think someone should be responsible for making sure the new engineer is set up all necessary calendar invites-- should the manager handle coordinating that?